tools/autograd/derivatives.yaml

```diff
@@ -221,6 +221,9 @@
 - name: exp(Tensor self)
   self: grad * result
 
+- name: expm1(Tensor self)
+  self: grad * self.exp()
```

Comment:
This should have been `grad * result`, like `exp` above it

Reply:
grad * result will be d(f(x)) * [exp(f(x)) - 1]. But here it is supposed to be d(f(x)) * [exp(f(x))]. This mistake was done earlier, which I fixed in a future commit.

Reply:
Ah right. Good point

Reply:
However, I think it would still be better to do `(result + 1)` - not only is it cheaper than exp, but also will avoid numerical problems (remember that expm1 is meant to be used when exp is not accurate enough, and this grad formula defeats the purpose)

Reply:
I can understand. Would you like me to make changes and commit?

Reply:
yes, please send a PR